.gitignore

```diff
@@ -19,6 +19,7 @@ flow-typed/npm/*
 
 # Release
 dist/*
+output/*
```

Comment:
Are you sure that we need to hide this folder? If yes, maybe we also don't need it to be published at npm? If we need it to be published at npm, we need to have this folder at `package.json -> "files" section`.

Reply:
It's not neccessary

Reply:
sure sure sure? why we have it then?